### DISCUSSION TITLE
Canva pro ?

### DISCUSSION THREAD

**Discussion by Prathamgupta661** [community/community#29408] [Other Feature Feedback, Questions, & Ideas]
Can anyone tell me how to connect the student developer pack with canva pro

**Comment by melvincwng** [comment]
@Prathamgupta661 , Hi there, if I'm not mistaken, as of now, Canva Pro isn't available in the GitHub Student Developer Pack.

Hope this clarifies.

**Reply by seapharms** [reply]
I thought it was available.

**Reply by melvincwng** [reply]
@seapharms, it used to be available.

But as of now, Canva Pro seems to have been removed from the list of benefits in the GitHub Student Developer Pack.

You can refer to this link for more information on what benefits are available still: https://education.github.com/pack

Hope this helps
